scout/export/variant.py

```diff
@@ -1,26 +1,26 @@
+from pprint import pprint as pp
 
 
 def export_causatives(adapter, collaborator):
-    """docstring for export_causatives"""
-    #put variants in a dict to get unique ones
-    variants = {}
-    
-    for variant in adapter.get_causatives(institute_id=collaborator):
-        variant_id = '_'.join(variant.variant_id.split('_')[:-1])
-        variants[variant_id] = variant
+    """Export causative variants
     
+    Args:
+        adapter(MongoAdapter)
+        collaborator(str)
+    """
+    #put variants in a dict to get unique ones
     variant_string = ("{0}\t{1}\t{2}\t{3}\t{4}\t{5}\t{6}\t{7}")
     
-    for variant_id in variants:
-        variant = variants[variant_id]
+    for document_id in adapter.get_causatives(institute_id=collaborator):
+        variant_obj = adapter.variant(document_id)
         yield variant_string.format(
```

Comment:
are all these fields garuanteed to exist?

Reply:
Jepp